src/api/payment-requests/payment-requests.md

```diff
@@ -94,9 +94,8 @@ external ledger. Once you have made that payment, you can use the transaction id
 
 An order item for which payment is requested. The currency and units for a line
 item price will be consistent with the payment request value and the sum of
-line item prices should equal the payment request value.
-
-A line item price can be negative to represents a discount.
+line item prices should equal the payment request value.  A line item price may
+be negative to represent a discount.
```

Comment:
How is tax treated on a negative line item?

Can you have negative tax if it is removing value from other line items?